### PR TITLE
[WIP] Add the ability to Jump to Cut  to also jump to keyframes

### LINKED ISSUE #220: Override jump to keyframe (⬆⬇) to also jump to cut, including during playback
The idea is to make it so the ⬆ and ⬇ jump to the previous and next cut, reliably (during playback, which blender's default tool doesn't support)

Optional idea: automatically select linked strips when moving to cut?

### PR DESCRIPTION
**Pull request by henriiquecampos**
This closes #220 although I dunno if it actually overrides. Any directions on how to do that override, @NathanLovato ?

### CODE REVIEW

File: operators/jump_to_cut.py [hunk 1]
```diff
@@ -16,8 +16,15 @@ class JumpToCut(bpy.types.Operator):
...
         'demo': '',
         'description': doc_description(__doc__),
-        'shortcuts': [],
-        'keymap': 'Frames'
+        'shortcuts': [
+            ({'type': 'UP', 'value': 'PRESS'},
+             {'direction': 'forward'},
+             'Jump to next cut or keyframe'),
+            ({'type': 'DOWN', 'value': 'PRESS'},
+             {'direction': 'forward'},
```
Comment (NathanLovato) [inline operators/jump_to_cut.py]: Both shortcuts go forward

**Comment by NathanLovato** [comment]
How come you don't know if it overrides the shortcut or not? You didn't test your code in Blender?

**Comment by NathanLovato** [comment]
On Windows if you register shortcuts they generally take precedence over the default ones, but not always.

**Comment by henriiquecampos** [comment]
> How come you don't know if it overrides the shortcut or not? You didn't test your code in Blender?

It didn't work for me, but I dunno what happens in actual setup (i.e. how it actually sets the shortcuts using what we set in this doc dict in __init__ or something). I also don't know how to actually override it through code.

So what I did was to manually set the shortcuts manually, and it didn't override, I had to remove the jump to keyframe shortcut.

File: operators/jump_to_cut.py (hunk 1 above)
Comment (henriiquecampos) [reply operators/jump_to_cut.py]: Fixed in 5b76092530d1b24a34f9d3b45de5f6cf384fc4fb 😄

**Comment by henriiquecampos** [comment]
I think it doesn't directly override the shortcut because we are using 2 different `'keymap': 'Sequencer'`. 

My bad, I changed it to Sequencer, instead of keeping it as Frames. It overrides now

**Comment by NathanLovato** [comment]
Thanks!

**Pull request merged**